products/extensions/b2b-components/index.md

```diff
@@ -20,3 +20,134 @@ In the world of digital B2B commerce, where businesses engage with other compani
 * **Quick Order and Shopping List** takes care of distinctive B2B buying behaviors.
 
 * **Digital Sales Composables** aims to provide a set of composable frontends to cover more complex Sales Representative jobs.
+
+## Configuring custom toggles for B2B components
+
+The B2B Components allow merchants to selectively choose and configure B2B capabilities according to their needs. They offer merchants the ability to craft a tailored B2B ecommerce experience for their business partners while also allowing agencies to fine-tune Shopware to meet specific requirements. This means that B2B Components can be individually activated or deactivated for each business partner within the shop.
+
+The following articles will guide you how to do this by creating custom toggles via a plugin for B2B Components (Customer-specific features).
+
+The **Customer-specific features** section on the Customer detail page allows the shop merchant to turn these B2B features on or off for a specific customer. 
+
+![Feature Toggles](../../../assets/b2b-feature-toggles.png)
+
+To achieve this, ACL and address the following cases where functionality may be hidden:
+
+1. If the merchant has not activated a feature for a particular customer, it should be hidden.
+2. If the B2B admin has not granted an employee access to a specific feature, it should not be visible.
+
+Considering these scenarios, we can ensure that the appropriate B2B features are displayed and accessible based on feature toggles and admin-granted permissions.
+
+### Prerequisite
+
+To improve organization and maintain a clear structure, it is advisable to relocate all B2B Components into the `B2B` folder within the Commercial plugin. By doing so, you can centralize the B2B-related functionality, making it easier to locate, manage, and maintain the codebase. This folder structure promotes better separation of concerns and enhances the overall modularity of the application.
+
+```
+├── src
+│   ├── B2B
+│   │   ├── QuickOrder
+│   │   ├── AnotherB2BComponent
+│   │   │   CommercialB2BBundle.php
```

Comment:
where is this "CommercialB2BBundle.PHP" fall. Is it under "AnotherB2BComponent" or elsewhere?
If its under "AnotherB2BComponent", then we need to mark a line there.

Reply:
It is the only file here, so it doesn't fall under "AnotherB2BComponent" or anywhere else.